src/repository/x509.rs

```diff
@@ -1065,7 +1065,7 @@ impl Validity {
         Validity { not_before, not_after }
     }
 
-    pub fn from_duration(duration: Duration) -> Self {
+    pub fn from_duration(duration: TimeDelta) -> Self {
```

Comment:
Do you also want to rename the functions and the arguments or do you want to leave them as is?

Reply:
I want to keep the name so as to not making this a breaking change.